modules/ding_library/ding_library.field_group.inc

```diff
@@ -127,103 +127,6 @@ function ding_library_field_group_info() {
   );
   $field_groups['group_ding_library_images|node|ding_library|form'] = $field_group;
 
-  $field_group = new stdClass();
```

Comment:
This I don't see a reason to do?

Reply:
Field groups are used to put certain fields into common containers, so I guess it was solely to aid theming. These groups were interfering.